recipes/systemd.rb

```diff
@@ -1,18 +1,18 @@
 #
-# Cookbook:: nexus-repository-manager
+# Cookbook:: nexus_repository_manager
 # Recipe:: systemd
 #
 # Copyright:: Copyright (c) 2017-present Sonatype, Inc. Apache License, Version 2.0.
 
-systemd_unit 'nexus-repository-manager.service' do
+systemd_unit 'nexus_repository_manager.service' do
```

Comment:
Is the service name a chef implementation? I think this may need to be reverted.

Reply:
Yes, this is a system name and I will revert it.